packages/shared/src/hooks/notifications/useStreakRecoverModal.ts

```diff
@@ -15,14 +15,16 @@ export const useStreakRecoverModal = (): void => {
 
   useEffect(() => {
     if (!user || !streakRestore) {
+      closeModal();
```

Comment:
We need to check also if the query has been fetched as this would result to always closing right away.